bin/run_epix

```diff
@@ -125,11 +130,22 @@ def main(args):
     result = result[ak.argsort(result['t'])]
     result['t'] = result['t'] - result['t'][:, 0]
     result = result[result['t'] <= args.MaxDelay]
-    # Secondly truly separate events by time (1.1 times the max time),
-    # with first event starting at max time (needed for wfsim)
-    dt = np.arange(1, len(result['t'])+1) + np.arange(1, len(result['t'])+1) / 10
-    dt *= args.MaxDelay
-    result['t'] = (result['t'][:, :] + result['t'][:, 0] + dt)
```

Comment:
Notice that, following a WFsim bug hunt with @zhut19 and @petergaemers (althought I don't remember now the exact issue), we decided to have the first processed event already carry the corresponding 1.1x delay. This needs to be fix for the current branch.

Reply:
I will change it to `dt = np.arange(1, n_events+1)+np.arange(1, n_events+1)/10` inside `clean_separation` to get in line with the separation in the master branch.